_docs/_latest/use/inventory.md

```diff
@@ -6,12 +6,15 @@ order: 002
 # {{ page.title }}
 
 Forseti Inventory collects and stores information about your Google Cloud Platform (GCP)
-resources. Forseti Scanner and Enforcer use Inventory data to perform operations.
+resources. The Inventory data is transformed to data model, which will be used
```

Comment:
Missing article and simplify "... to a data model that is used by..."

Reply:
done